protractor/e2e/integretion/second-page.spec.ts

```diff
@@ -55,9 +57,22 @@ describe('test for second page', () => {
         await secondPage.waitForElementVisible(secondPage.pdfContent);
     });
 
-    fit('info card error case', async () => {
+    it('info card error case', async () => {
         await secondPage.scrollToElement(secondPage.infoCardSection);
         await secondPage.waitForElementVisible(secondPage.infoEmailInpt);
         await secondPage.infoSubmitBtn.click();
     });
+
+    fit('should open page blog pages with articles', async () => {
+        await secondPage.waitForElementVisible(secondPage.blogHeaderLink);
+        await secondPage.blogHeaderLink.click();
+        await blogPage.waitForElementVisible(blogPage.postClass.first());
+        expect(await blogPage.getCurrentUrl()).toContain(blogPage.pageUrl);
+        expect(await blogPage.postClass.count()).toEqual(20);
```

Comment:
4. And user see 20 articles with header, description and link - this is your task
in your test you check only amount of posts, it doesn't mean, that each article has header, link, etc. So, will be good to have something like these checks:
        const expectedArticlesLength = 20;
       expect(await $$('article').count()).toBe(expectedArticlesLength);
        expect(await $$('article header').count()).toBe(expectedArticlesLength);
        expect(await $$('article .post-entry').count()).toBe(expectedArticlesLength);
        expect(await $$('article a[class*="btn-action"]').count()).toBe(expectedArticlesLength);